modules/npm/src/main/typescript/main.d.ts

```diff
@@ -1,3 +1,8 @@
-declare class Foo {
-  constructor();
+declare class PlotPoint {
+  constructor(instant: Date, airmass: Number, altitude: Number);
+  airmass: Number;
+  altitude: Number;
+  instant: Date;
 }
+
+declare function nightPlot(site: Site, start: Date, coords: String): PlotPoint[];
```

Comment:
It should be `number` instead of `Number` and `string` instead of `String`.

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types

Reply:
Uh thanks for noticing that

Reply:
Also I wonder if it should be `export declare class` and `export declare function`? I'm not sure 🤔

Reply:
It is working as:
```typescript
import { GS } from 'lucuma-core'
import { PlotPoint } from 'lucuma-core'
import { nightPlot } from 'lucuma-core'

const points: Array<PlotPoint> = nightPlot(GS, new Date(), "09 55 33.173 +69 03 55.06")

console.log(points.length)
```

Reply:
Also note that I have a `Site` param which isn't defined!

Reply:
Does that code pass the typescript compiler?